### ISSUE TITLE
Script focuses player when pressing any link on yt

### ISSUE THREAD

**Issue by lihuelworks** [GregHerendi/YoutubeKeysFix#1]
Hi. I don't know if the proyect is being mantained, but the script has been incredibly helpful, and almost essential to my Youtube usage. Thing is, I've encountered a problem when choosing certain links, for example:

- the "view more" buttons on comments
- the "view X replies" in comments
- adding a video to a playlist from the sidebar
- adding a video to watch later from the sidebar

I don't know what can be the cause of this, or if you can replicate it; but if anything can be done or if there is any way I can help debugging this, please let me know.

**Comment by lihuelworks** [comment]
Ok, reading the script a little bit better, I think it's intended behavior. The onMouse() function specifies an event where "clicking any outside of an area" focuses the player:
```
    function onMouse(event) {
        // Called when mouse button is pressed over an element.
        // Debug log of mouse button event
        //console.log("YoutubeKeysFix: " + event.type + " ->", event.target);

        // click OUTSIDE OF AREAS focuses player
        //if (0 === getAreaOf(event.target))  return redirectFocus(event, playerElem);
    }
```
~~Commenting as in the snippet seems to correct the behavior~~. Since the script hasn't been updated in a while (Not in an accusatory manner, life happens), it's possible that the Youtube DOM has suffered some changes that made that behavior trigger for any click; or the definition of an "outside area", in the terms it was first described, changed. 

Edit: Well, seems I haven't read it enough. Disabling the "if" doesn't make the script not work, but you have to click before it takes (which deletes the purpose of the script). I also forgot you cannot reopen your own closed issues, [for some reason](https://github.com/isaacs/github/issues/583).

**Comment by Calciferz** [comment]
Thank you for the report and apologies for not maintaining the project for so long.

7111ccfb6367e3f45f2edc8676cc0ea25c8eee6f fixes this by removing the odd behavior focusing the player.
2df9e31c42cb72667064048093773ae4d9160962 fixes the misidentification of the comments section.

Your comment https://github.com/Calciferz/YoutubeKeysFix/issues/1#issuecomment-662860511 was very insightful, much appreciated!

v1.2.0 fixes this: 7111ccfb6367e3f45f2edc8676cc0ea25c8eee6f
